library_generation/test/integration_tests.py

```diff
@@ -42,15 +44,24 @@
     "google-cloud-java": "chore/test-hermetic-build",
     "java-bigtable": "chore/test-hermetic-build",
 }
-config_dir = f"{script_dir}/resources/integration"
 baseline_config_name = "baseline_generation_config.yaml"
 current_config_name = "current_generation_config.yaml"
 
+# This variable is used to override the jar created by building the image
+# with our own downloaded jar in order to lock the integration test to use
+# a constant version specified in
+# library_generation/test/resources/integration/test_generator_coordinates
+# This allows us to decouple the generation workflow testing with what the
+# generator jar will actually generate.
+# See library_generation/DEVELOPMENT.MD
```

Comment:
```suggestion
# See library_generation/DEVELOPMENT.md
```

Reply:
Could you specify which part of `development.md` you are refering?

Reply:
Done.